packages/@webex/plugin-authorization-browser-first-party/src/authorization.js

```diff
@@ -240,6 +242,128 @@ const Authorization = WebexPlugin.extend({
       });
   },
 
+  /**
+   * Get an OAuth Login URL for QRCode. Generate QR code based on the returned URL.
+   * @instance
+   * @memberof Credentials
+   * @returns {Object}
+   */
+  getQRCodeLoginDetails() {
+    return this.webex
+      .request({
+        method: 'POST',
+        service: 'oauth-helper',
+        resource: '/actions/device/authorize',
+        form: {
+          client_id: this.config.client_id,
+          scope: this.config.scope,
+        },
+        auth: {
+          user: this.config.client_id,
+          pass: this.config.client_secret,
+          sendImmediately: true,
+        },
+      })
+      .then((res) => {
+        return res.body;
+      })
+      .catch((res) => {
+        if (res.statusCode !== 400) {
+          return Promise.reject(res);
+        }
+
+        const ErrorConstructor = grantErrors.select(res.body.error);
+
+        return Promise.reject(new ErrorConstructor(res._res || res));
+      });
+  },
+
+  /**
+   * Polling the server to check whether the user has completed authorization
+   * @instance
+   * @memberof Credentials
+   * @param {Object} options
+   * @returns {Object}
+   */
+    startQRCodePolling(options = {}) {
+      if (!options.device_code) {
+        return Promise.reject(new Error('A deviceCode is required'));
+      }
+  
+      if (this.pollingRequest) {
+        return Promise.reject(new Error('There is already a polling request'));
+      }
+  
+      const {device_code: deviceCode, interval = 2, expires_in: expiresIn = 300} = options;
+  
+      let attempts = 0;
+      const maxAttempts = expiresIn / interval;
+  
+      return new Promise((resolve, reject) => {
+        this.pollingRequest = setInterval(() => {
+          attempts += 1;
+  
+          this.webex
+            .request({
+              method: 'POST',
+              service: 'oauth-helper',
+              resource: '/actions/device/token',
+              form: {
+                grant_type: 'urn:ietf:params:oauth:grant-type:device_code',
+                device_code: deviceCode,
+                client_id: this.config.client_id,
+              },
+              auth: {
+                user: this.config.client_id,
+                pass: this.config.client_secret,
+                sendImmediately: true,
+              },
```

Comment:
I do not think you need the `auth` param for this request. Could you please check and link the docs if it is needed.

Reply:
Same as above